dspace/config/emails/healthcheck

```diff
@@ -3,7 +3,7 @@
 ## Parameters: {0} is the output of healthcheck command
 ## See org.dspace.core.Email for information on the format of this file.
 ##
-#set($subject = 'Repository healthcheck')
+#set($subject = "${config.get('dspace.name')}: Repository healthcheck')
```

Comment:
I'm sorry that I missed this earlier:  the final single-quote should be a double-quote.  This template causes a ParseErrorException in Velocity.

Reply:
After this correction, I can approve.  The emails that are sent appear to be correct and making good use of the relevant configuration properties.